walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -25,22 +25,61 @@
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     Set<TimeRange> busyTimes = new HashSet<>();
```

Comment:
nit: for consistency of naming either:
mandatoryAttendeesEventTimes
optionalAttendeesEventTimes
or 
mandatoryBusyTimes
optionalBusyTimes 
 <or other consistent equivalent>